sqlglot/generator.py

```diff
@@ -90,6 +92,8 @@ class Generator:
         "time_trie",
         "pretty",
         "configured_pretty",
+        "quote_start",
+        "quote_end",
```

Comment:
delete quote from args and the doc string?

Reply:
and slots

Reply:
Fixed